distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -171,6 +174,50 @@ type Vector a
     new_builder : Integer -> Builder
     new_builder (capacity=10) = Builder.new capacity
 
+    ## PRIVATE
+       ADVANCED
+       Creates a new `Vector` by providing a `Builder` to the provided function.
```

Comment:
I'd rephrase this sentence as _providing X to the provided Y_ sounds weird.

Reply:
Changed to "passing a `Builder` to the provided function"